php-practice/section6.php

```diff
@@ -164,9 +191,18 @@ function getTriangleNumber($x) {
  * @return int
  */
 function sum1To100Evens() {
-  // YOUR CODE HERE
+  $value = 0;
+
+    for($i = 0; $i < 100; $i+=2) {
+      $value += $i;
+    }
+
+    return $value;
 }
 
+echo sum1To100Evens().PHP_EOL;
```

Comment:
As a bonus question, could you implement the same function using the modulus operator?

[The PHP docs on modulus are here](http://php.net/manual/en/language.operators.arithmetic.php].

Basically, modulus returns the **remainder** of the number on the left divided by the number on the right.

```php
echo 5 % 2; // 1
echo 10 % 7; // 3
echo 21 % 7; // 0
echo 101 % 10; // 1
```

Make sense?

So if I do:

```php
function sum1To100Evens() {
  $value = 0;

  for ($i = 0; $i <= 100; $i++) {
    if (/* YOUR CODE HERE */) {
      $value += $i;
    }
  }

  return $value;
```

What condition would you put in that `if` statement in order to check if `$i` is even? (Again, using the `%` operator.)